qiskit_ibm_runtime/execution_span/execution_spans.py

```diff
@@ -113,3 +116,22 @@ def sort(self, inplace: bool = True) -> "ExecutionSpans":
         obj = self if inplace else ExecutionSpans(self)
         obj._spans.sort()
         return obj
+
+    def draw(self, normalize_y: bool = False) -> "PlotlyFigure":
+        """Draw these execution spans on a plot.
+
+        .. note::
+            To draw multiple sets of execution spans at once, for examule, coming from multiple
+            jobs, consider calling :func:`~.draw_execution_spans` directly.
```

Comment:
```suggestion
            To draw multiple sets of execution spans at once, for example coming from multiple
            jobs, consider calling :func:`~.draw_execution_spans` directly.
```

Reply:
In the swarm plot that I'm working on I [solved it like this](https://github.com/Qiskit/qiskit-ibm-runtime/pull/1988/files#diff-6b933f9efdba754913c7aa6a601a7d4251cfaeb8c9141628175f1d761337f89dR384). Choose which one you like the most ("consider calling this other func" or "other_spans") and I will make my plotter consistent